### PR TITLE
tools to find breakpoints and to find ubiquitous reference kmers

### PR DESCRIPTION
**Pull request by tedsharpe**
note that this PR is onto sv_master, not onto master

### CODE REVIEW

**Comment by lbergelson** [comment]
@tedsharpe Is this for general review or just for @cwhelan?

**Comment by cwhelan** [comment]
So, as we muck around with more exploratory code, we've created a branch
sv_master and are going to make feature branches and pull requests to that,
that we'll give light code review to among ourselves but more use for
sharing code. Then when we are ready to submit working code into the master
gatk repo, we'll clean it up and submit it for more rigorous review, and
probably have a gatk4 team member review the spark/infrastructure parts.
Does that plan sound ok with you?

**Comment by tedsharpe** [comment]
Feel free to watch us think out loud, but this stuff isn't ready for 
prime time.  PR is for some sketchy code to get merged by SV team into 
sv_master.

File: build.gradle [hunk 1]
```diff
@@ -39,6 +39,7 @@ task downloadGsaLibFile(type: Download) {
 
 repositories {
     mavenCentral()
+    mavenLocal()
```
Comment (cwhelan) [inline build.gradle]: Just curious, what this is for?

**Comment by lbergelson** [comment]
@cwhelan That plan seems ok.  I think it's probably best to merge to master as soon as possible though.  Short cycles with small pull requests are definitely easier for everyone.  If you leave it separate for a long time it's going to be very painful to rebase / review.

File: build.gradle (hunk 1 above)
Comment (tedsharpe) [inline build.gradle]: It allows one to build htsjdk and pick up the updated artifact from your 
local maven repo.

**Pull request merged**